GDJS/GDJS/IDE/ExporterHelper.cpp

```diff
@@ -261,16 +265,37 @@ bool ExporterHelper::ExportCordovaFiles(const gd::Project &project,
           .FindAndReplace("<!-- GDJS_ICONS_ANDROID -->", makeIconsAndroid())
           .FindAndReplace("<!-- GDJS_ICONS_IOS -->", makeIconsIos());
 
-  if (!project.GetAdMobAppId().empty()) {
-    str = str.FindAndReplace(
-        "<!-- GDJS_ADMOB_PLUGIN_AND_APPLICATION_ID -->",
-        "<plugin name=\"cordova-plugin-admob-free\" spec=\"~0.21.0\">\n"
-        "\t\t<variable name=\"ADMOB_APP_ID\" value=\"" +
-            project.GetAdMobAppId() +
-            "\" />\n"
-            "\t</plugin>");
+  gd::String plugins = "";
+
+  for(std::shared_ptr<gd::PlatformExtension> extension : project.GetCurrentPlatform().GetAllPlatformExtensions()) { //TODO Add a way to select only used Extensions
+    for (gd::DependencyMetadata dependencies: extension->GetAllDependencies()) {
+      if (dependencies.GetDependencyType() == "cordova") {
+
+        plugins += "<plugin name=\"" + dependencies.GetExportName();
+        if(dependencies.GetVersion() != "-1") {
```

Comment:
As written in another comment, can we just use the empty string instead of "-1"?

Reply:
Oh sorry I must have missed it